### ISSUE TITLE
In-App Messaging still in beta?

### ISSUE THREAD

**Issue by saldous** [firebase/firebase-ios-sdk#13664]
### Description

When I do pod install with pod '**FirebaseInAppMessaging**' in my podfile, it installs v0.13.0

I found a closed ticket from almost a year ago saying it is in beta and to use **pod 'FirebaseInAppMessaging', '> 10.13-beta'**, which installs 11.2.0-beta. But your documentation doesn't say to do this? https://firebase.google.com/docs/in-app-messaging/get-started?platform=ios

I want to make sure I'm getting the right version. Thanks.

### Reproducing the issue

Pod install with pod '**FirebaseInAppMessaging**' in podfile installs v0.13.0

### Firebase SDK Version

11.2

### Xcode Version

15.4

### Installation Method

CocoaPods

### Firebase Product(s)

In-App Messaging

### Targeted Platforms

iOS

### If using CocoaPods, the project's Podfile.lock

  pod 'FirebaseAnalytics'
  pod 'FirebaseAuth'
  pod 'FirebaseCore'
  pod 'FirebaseCrashlytics'
  pod 'FirebaseFirestore'
  pod 'FirebaseMessaging'
  pod 'FirebaseInAppMessaging'
  pod 'FirebaseRemoteConfig'

**Comment by paulb777** [comment]
Yes the In App Messaging SDK is still in beta. There's an open issue at https://github.com/firebase/firebase-ios-sdk/issues/12810 tracking the documentation problem.

I'll close that one in favor of this one, since it's clearer. 

To summarize, use `pod 'FirebaseInAppMessaging', '> 11.2-beta'` while the product is in beta.

**Comment by kondayutarou** [comment]
I'm wondering why exactly Firebase In-App messaging is still in beta.
Is there any functionality that still isn't perfected? Or is it a reliability issue?
